code/modules/client/preferences.dm

```diff
@@ -105,7 +105,16 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 		load_path(parent.ckey)
 		if(load_and_save && !fexists(path))
 			try_savefile_type_migration()
-		unlock_content = !!parent.IsByondMember()
+		var/byond_member = parent.IsByondMember()
+		if(isnull(byond_member)) // Connection failure, retry once
+			byond_member = parent.IsByondMember()
+			var/static/admins_warned = FALSE
+			if(!admins_warned)
+				admins_warned = TRUE
+				message_admins("Byond membership lookup had a connection failure for a user. This is most likely an issue on the byond side but if this consistently happens you should bother your server operator to look into it.")
```

Comment:
Perhaps this should also be logged with log_admin or log_game so that if someone wants to check game logs after the fact if it happened, it will be recorded.

Reply:
log_admin is logging for admin actions but yeah I can stick this in log_game